### ISSUE TITLE
rating goes up by 0.5

### ISSUE THREAD

**Issue by zigurana** [zigurana/EmulationStation#6]
easy: https://github.com/jgeumlek/EmulationStation-tweaked/commit/fccdd44ac4a69890e15c554f8c34f46789fe773f

**Comment by zigurana** [comment]
nonsense

**Comment by robertybob** [comment]
Why?

**Comment by zigurana** [comment]
:-) too cryptic? ok, let me rephrase:
non-issue

(translation: there are much more important issues to tackle in ES, if this is needed, lets implement it as a PR to retropie-master directly, I  want it of my list of things to do)
